README.md

```diff
@@ -47,37 +41,154 @@ So that I am reassured that my order will be delivered on time
+
+
+## Living Documentation
+
+Generated by using rspec with current test suite
+
+```rs
+Dish
+  #name
+    returns the stored name
+  #price
+    returns the stored price
+  #available?
+    defaults to true
+  #set_unavailable
+    causes available? to return false
+  #set_available
+    after having been set to set_unavailable
+      causes available? to return true
+
+Menu
+  #add_dish
+    updates the list of items
+  #list_dishes
+    returns a formatted list of dishes
+      with a single dish
+      with multiple dishes
+  #find_by_name
+    returns an item that matches the name given
+  #list_available_dishes
+    returns a formatted list of available dishes
+
+Order
+  #add_to_order
+    adds an item to the user's order
+  #remove_from_order
+    if item is in current order
+      removes the item
+    if item is not in current order
+      raises an error
+  order contains three pizzas
+    #order_total_price
+      returns the sum of the item prices
+    #list_order
+      prints all items in the order
+    #select_items
+      when given a valid item
+        adds the dish to the order
+
+Takeaway
+  #send_confirmation_text
+    sends a confirmation text to TwilioManager
+
+TwilioManager
+  #initialize
+    sets up a new Twilio Client
+
+Finished in 1.77 seconds (files took 0.72895 seconds to load)
+18 examples, 0 failures
+```
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+## Credentials File
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+In order to use this program you will need a Twilio account (free trial available) and once set up you will also need to populate a 'credentials.rb' in the root of the repository with your information. Below is an example.
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+```
+AUTH_TOKEN = Your Twilio Authorization Token (Single-quoted string)
+ACCOUNT_SID = Your Twilio Account SID (Single-quoted string)
+MESSAGING_SID = The Twilio Messaging service SID (Single-quoted string)
+TARGET_PHONE = '+441111111111' (Single-quoted string)
+```
+
+## Example Use in IRB
+
+Load `irb` in the root directory then run the following code snippet:
+
+```irb
+Dir['lib/*.rb'].each { |file| load file }
+dishes = [
+  Dish.new('Pizza', 16),
+  Dish.new('Sushi', 12),
+  Dish.new('Fish & Chips', 9),
+  Dish.new('Thai Green Curry', 14),
+  Dish.new('Chicken Wings', 7)
+]
+menu = Menu.new
+dishes.each { |dish| menu.add_dish(dish) }
+twilio = TwilioManager.new
+order = Order.new
+takeaway = Takeaway.new(twilio, menu, order)
+takeaway.interactive_menu
+```
```

Comment:
Very detailed instructions for use, I must emulate this

Reply:
Thank you, it was also very helpful for me running it quickly in irb without having to manually type multiple lines of code!